### ISSUE TITLE
Module VEP broke

### ISSUE THREAD

**Issue by quentin67100** [ewels/MultiQC#1597]
### Description of bug

I tried to use MultiQC with singularity. But it failled to add VEP stats. I also tried with the conda version of multiqc but it failled too.

### File that triggers the error

[Pologne.stat.vep.html.zip](https://github.com/ewels/MultiQC/files/7608793/Pologne.stat.vep.html.zip)
,

### MultiQC Error log

```console
/// MultiQC 🔍 | v1.11

|           multiqc | Search path : /shared/projects/gentaumix/Pologne
|          qualimap | Found 4 BamQC reports
|    MarkDuplicates | Skipping MarkDuplicates sample 'multiqc_data' as missing essential fields
|    MarkDuplicates | Skipping MarkDuplicates sample 'multiqc_data' as missing essential fields
|    MarkDuplicates | Skipping MarkDuplicates sample 'multiqc_data' as missing essential fields
|    MarkDuplicates | Skipping MarkDuplicates sample 'multiqc_data' as missing essential fields
|    MarkDuplicates | Skipping MarkDuplicates sample 'multiqc_data' as missing essential fields
|            picard | Found 4 MarkDuplicates reports
|            picard | Found 8 ValidateSamFile reports
╭────────────────── Oops! The 'vep' MultiQC module broke... ───────────────────╮
│ Please copy this log and report it at                                        │
│ https://github.com/ewels/MultiQC/issues                                      │
│ Please attach a file that triggers the error. The last file found was:       │
│ /shared/projects/gentaumix/Pologne//06_VEP/Pologne.stat.vep.html             │
│                                                                              │
│ Traceback (most recent call last):                                           │
│   File "/usr/lib/python3.8/site-packages/multiqc/multiqc.py", line 624, in r │
│     output = mod()                                                           │
│   File "/usr/lib/python3.8/site-packages/multiqc/modules/vep/vep.py", line 3 │
│     self.parse_vep_html(f)                                                   │
│   File "/usr/lib/python3.8/site-packages/multiqc/modules/vep/vep.py", line 1 │
│     existing = values[1].split("(")[0].replace(" ", "")                      │
│ IndexError: list index out of range                                          │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
|      fastq_screen | Found 8 reports
|            fastqc | Found 8 reports
|           multiqc | Compressing plot data
|           multiqc | Report      : ../../../../../projects/gentaumix/Pologne/Pologne_MultiQC_global.html
|           multiqc | Data        : ../../../../../projects/gentaumix/Pologne/Pologne_MultiQC_global_data
|           multiqc | MultiQC complete
```

**Comment by ewels** [comment]
@maxulysse - are you interested in taking a look at this?

**Comment by PROJECT-DEFIANT** [comment]
I had similar issue with v.1.11 from bioconda but with the output from annotating empty vcf file

`QC module broke...
================================================================================
Traceback (most recent call last):
  File "/home/szszyszkowski/.local/lib/python3.8/site-packages/multiqc/multiqc.py", line 651, in run
    output = mod()
  File "/home/szszyszkowski/.local/lib/python3.8/site-packages/multiqc/modules/vep/vep.py", line 37, in __init__
    self.parse_vep_html(f)
  File "/home/szszyszkowski/.local/lib/python3.8/site-packages/multiqc/modules/vep/vep.py", line 131, in parse_vep_html
    existing = values[1].split("(")[0].replace(" ", "")
IndexError: list index out of range`

**Comment by maxulysse** [comment]
@ewels I'd be happy to have a look, sorry for the late reply

**Comment by ewels** [comment]
Hi all,

This should now be fixed in 1977df64c13823751853e3b8e3246b1059abf16d - table cells containing `-` are not reported and will be empty in the MultiQC table now (the entire column will be missing if all samples had this). The module should no longer crash.

Thanks for reporting and providing an example report 👍🏻 

Phil
